src/components/DisplayedNameChangeController/DisplayedNameChangeController.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { FormInput } from '../inputs/FormInput/FormInput';
+import { useAPIUpdateUserInfo } from '@src/api/users/users';
+import { AxiosError } from 'axios';
+import { UPDATE_USER_INPUT_DELAY_TIME } from '@src/constants/userSettings';
+import Toast from 'react-native-toast-message';
+
+type DisplayedNameChangeControllerProps = {
+	displayedName?: string;
+};
+
+export const DisplayedNameChangeController = ({ displayedName }: DisplayedNameChangeControllerProps) => {
+	const { t } = useTranslation();
+
+	const [name, setName] = useState(displayedName || '');
+
+	useEffect(() => {
+		const timer = setTimeout(() => {
+			if (name) {
+				updateUserInfo({ data: { alias: name } });
+			}
+		}, UPDATE_USER_INPUT_DELAY_TIME);
+
+		return () => clearTimeout(timer);
+	}, [name]);
+
+	const { mutate: updateUserInfo } = useAPIUpdateUserInfo<AxiosError, unknown>({
+		mutation: {
+			onSuccess: (data) => {
+				setName(data.data.alias);
+			},
+			onError: (error) => {
+				Toast.show({ type: 'error', text1: 'Failed to save name' });
+				console.error('Error updating name:', error);
```

Comment:
redundant console logs

Reply:
Console error may cause app to break in production. Overall, please do not let console logs get in your prod app. You won't be able to see these logs anyway, and they reduce app speed dramatically